### ISSUE TITLE
New Cert

### ISSUE THREAD

**Issue by larsenv** [mackworth/cTiVo#508]
Does cTiVo need to be updated with a new cert because it expired yesterday?

**Comment by mackworth** [comment]
Correct. You can [download it here](https://github.com/mackworth/cTiVo/releases/) : Use 3.5.3 unless you’re on an older OS

Thanks for the reminder, BTW, I forgot to push the button to push the upgrade.

**Comment by mackworth** [comment]
I've pushed the update, so if you haven't downloaded it yet, then you should be able to go to the menu item `cTiVo>Check For Updates...`. Let me know if you have any problems.
